### ISSUE TITLE
Fonts are not being bundled (Nuxt 3.6)

### ISSUE THREAD

**Issue by thijsw** [nuxt-community/google-fonts-module#137]
When I use the option to download fonts, they work during development, but it seems like they are not being bundled when I build the project.

Please note that this issue only occurs with the newest version of Nuxt, specifically version 3.6, which was just released.

**Comment by theguriev** [comment]
Same issue here :(

**Comment by benoit-ponsero** [comment]
`fontsPath: '/fonts'`
in nuxt.config seems do the trick. 

Anyway fonts are copied in `/` & `/fonts`.

**Comment by episage** [comment]
`fontsPath: '/fonts'` does the trick!

**Comment by cschweda** [comment]
Question: do you put the ```fontsPath: '/fonts'``` into the ```googleFonts``` section in ```nuxt.config```?

```
  googleFonts: {
       ...
       fontsPath: '/fonts'
       ...
   }
```

Or in the root of the nuxt config?

```
    export default defineNuxtConfig({
    
       ...
       fontsPath: '/fonts'
       ...

     })
```

**Comment by thijsw** [comment]
This seems to work for Google fonts, thanks!

However, it appears that other custom fonts in Nuxt 3.6 are also not getting bundled. I have created an issue here: https://github.com/nuxt/nuxt/issues/21784.

**Comment by theguriev** [comment]
> `fontsPath: '/fonts'` in nuxt.config seems do the trick.
> 
> Anyway fonts are copied in `/` & `/fonts`.

It works! Thank you! ❤️

**Comment by hartmut-co-uk** [comment]
(PS: the workaround causes quite the deterioration of CLS on page reload)

Luckily the workaround is no longer necessary with latest nuxt release 🚀 
https://github.com/nuxt/nuxt/releases/tag/v3.6.1

For me, all issues are resolved after the upgrade!

**Comment by ricardogobbosouza** [comment]
Fixed on nuxt v3.6.1
